pkg/apis/triggers/v1alpha1/event_listener_validation.go

```diff
@@ -49,6 +49,10 @@ func (s *EventListenerSpec) validate(ctx context.Context) *apis.FieldError {
 }
 
 func (t *EventListenerTrigger) validate(ctx context.Context) *apis.FieldError {
+	if t.Template == nil && t.TriggerRef == "" {
```

Comment:
as per the doc it shows that as part of `trigger` one can specify either `template`, `bindings` or `triggerRef`
So i think we need to add check for `bindings` also ?

Reply:
I think we need to phrase this better in the documentation.
What I meant was either (bindings`, `template` and `interceptors) or `triggerRef`.

Reply:
Ya that would be more clear :+1:

Reply:
I added this in documentation: A `trigger` field must have `template` (along with needed `bindings` and `interceptors`) or `triggerRef`.

Is this clear?

Reply:
Yep this is clear :+1: